grl/environment/spec.py

```diff
@@ -65,3 +66,12 @@ def load_spec(name, **kwargs):
     spec['phi'] = normalize(spec['phi'])
 
     return spec
+
+def load_pomdp(name: str, rand_key: np.random.RandomState = None, **kwargs) -> AbstractMDP:
+    """
+    Wraps a MDP/POMDP specification in a POMDP
+    """
+    spec = load_spec(name, rand_key=rand_key, **kwargs)
+    mdp = MDP(spec['T'], spec['R'], spec['p0'], spec['gamma'], rand_key=rand_key)
+    amdp = AbstractMDP(mdp, spec['phi'])
```

Comment:
I wonder if we should just rename `AbstractMDP` to `POMDP` now

Reply:
I was thinking the exact same... I'm just going to pull the trigger